samples/cellular/modem_shell/src/ping/icmp_ping.c

```diff
@@ -414,8 +414,9 @@ static uint32_t send_ping_wait_reply(struct icmp_ping_shell_cmd_argv *ping_args,
 	} while (true);
 
 	if (rai == PING_RAI_LAST_PACKET) {
+		int rai = RAI_NO_DATA;
```

Comment:
Maybe name this `rai_option` to align with the other variable related to SO_RAI.